### PR TITLE
uploading file Pratik

### PR DESCRIPTION
**Pull request by Pratik-patil-77**
hi

### CODE REVIEW

File: Pratik/hi [hunk 1]
```diff
@@ -0,0 +1 @@
+hi all
```
Comment (Pawan1005) [inline Pratik/hi:1]: add more content

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
add more

**Comment by Pratik-patil-77** [comment]
hi all
this is Pratik Patil
I'm From Pune

**Review by Pratik-patil-77 — COMMENTED** [review]
OK

**Pull request merged**